setup.py

```diff
@@ -40,7 +40,7 @@ def load_about():
     packages=find_packages(exclude=["tests*"]),
     include_package_data=True,
     python_requires=">=3.8",
-    install_requires=["tutor>=16.0.0,<17.0.0"],
+    install_requires=["tutor>=16.1.2,<17.0.0"],
```

Comment:
Note to self: this PR is dependent on [this one #901](https://github.com/overhangio/tutor/pull/901) and will require a new release of tutor.